draft-ietf-core-resource-directory.md

```diff
@@ -873,7 +873,7 @@ Failure:
 : 4.00 "Bad Request" or 400 "Bad Request". Malformed request.
 
 Failure:
-: 4.09 "Conflict" or 409 "Conflict". Attempt to update the registration content with links resulting in plurality of references; see {{link-plurality}}.
+: 4.09 "Conflict" or 409 "Conflict". Attempt to create a registration resource with ep and d value pair which is already present in an earlier created registration resource.
```

Comment:
I think that 4.09 can not occur at a registration any more at all; registering again does not result in a conflict, but simply in a 2.01 Created with the Location of the old registration, and otherwise the effects as if the same request had been a PUT to the registration resource. (Especially, lifetime counting starts anew.)